packages/toast/src/Toast.ts

```diff
@@ -116,28 +116,38 @@ export class Toast extends FocusVisiblePolyfillMixin(SpectrumElement) {
         return this._variant;
     }
 
+    @property({ type: String })
+    public set label(label: string) {
+        this._label = label;
+        this.setAttribute('label', label);
+        this.requestUpdate('label', label);
+    }
+
+    public get label(): string {
+        return this._label;
+    }
+
     private _variant: ToastVariants = '';
 
-    private renderIcon(variant: string): TemplateResult {
+    private _label: string = '';
```

Comment:
not sure but seems like changing the default string to be an empty string as opposed to the existing `Success`, `Error`, and `Information` seems like it could break things?

Reply:
Agreed, if the consumer does not define a label, should we fall back to the existing strings?
```suggestion
    <sp-icon-info label=${label === "" ? "Information" : label} class="type"></sp-icon-info>
```

Reply:
Yeah falling back to the default makes sense to me!